doc/JSON_FLAGS.md

```diff
@@ -250,6 +250,12 @@ Flags used to describe monsters and define their properties and abilities.
 - ```CBM_OP``` May produce a cbm or two from bionics_op when butchered.
 - ```FISHABLE``` It is fishable.
 - ```INTERIOR_AMMO``` Monster contains ammo inside itself, no need to load on launch. Prevents ammo from being dropped on disable.
+- ```INK_SAC``` When butchered, monster drops a number of ink sacs.
+- ```MUT_ABLE``` Allows for you pet monster to be injected with mutagen.
+- ```MUTANT``` Tells the game that the monster may be injected with purifier.
+- ```TENTACLES2``` When butchered, monster drops 2 tentacles. Also, controls how much attacks the TENTACLE_LASH does.
```

Comment:
Should be "how many", not "how much".
"Much" is for uncountable values, like volume or weight.